### DISCUSSION TITLE
Locally cache on iOS app

### DISCUSSION THREAD

**Discussion by mehapps** [ente-io/ente#3406] [Enhancements]
Hey all, i have a few (i mean like less than 100) videos/photos I sync to ente, have more than enough storage on iOS (1TB) so would like an option to cache them locally for faster loading rather than pulling from cloud every time. Would this be possible for implement?

**Comment by ua741** [comment]
The app already maintains a cache of items that you have tried to view/download.

**Reply by radh21301** [reply]
OP is probably asking for an option to cache all images? Not necessary the ones we tried to open or download.

Unless ente is already doing that, it will be really nice to have this. Maybe somewhere in the settings pane, there can be some statistics on the storage used by those etc.

**Reply by mehapps** [reply]
Yeah that's what I meant lol
